src/test/components/CallTreeSidebar.test.js

```diff
@@ -183,7 +183,7 @@ describe('CallTreeSidebar', function() {
     const layoutCategory = getByText('Layout');
     fireFullClick(layoutCategory);
 
-    getByText('FakeSubCategoryC');
+    expect(getByText('FakeSubCategoryC')).toBeTruthy();
```

Comment:
This is not strictly necessary, but the article https://kentcdodds.com/blog/common-mistakes-with-react-testing-library recommends this, and I believe this is a good idea.

Reply:
It's at least clear now that it's an assertion.